index.html

```diff
@@ -54,14 +54,80 @@ <h1>find your movie</h1>
 						<a class="c-menu__link" href="#">fantasy</a>
 					</li>
 				</ul>
+				<div class="c-select">
+					<label class="c-select__label">
+						<select class="c-select__bar">
+							<option>release date up</option>
+							<option>release date</option>
+							<option>vote average up</option>
+							<option>vote average</option>
+						</select>
+					</label>
+				</div>
 			</div>
-
-			<!--			<form>-->
-			<!--				<select class="c-select">-->
-			<!--					<option>release date</option>-->
-			<!--					<option>vote average</option>-->
-			<!--				</select>-->
-			<!--			</form>-->
+			<p class="c-main-layout__main-number-of-films"><span>6</span> movies found</p>
+			<ul class="c-main-layout__main-cards">
+				<li>
+					<div class="c-card">
+						<img src="../pictures/galaxy.jpg" alt="movie picture">
+						<div class="c-card__info u-clearfix">
```

Comment:
utility classes are better to use on elements without any other classes.
And it seems to me you can just set some "min-height" instead of clearfix styles